com.github.ADBeveridge.Raider.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "com.github.ADBeveridge.Raider",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "raider",
+    "rename-icon": "com.github.ADBeveridge.Raider",
+    "rename-appdata-file": "com.github.ADBeveridge.Raider.appdata.xml",
+    "rename-desktop-file": "com.github.ADBeveridge.Raider.desktop",
```

Comment:
these should be the original name. If the name is already correct, the commands should be removed.

(this is why it's failing)